### ISSUE TITLE
2.8.0 (final) release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2898]
## Since 2.8.0-rc1
- #2863 
- #2873 
- #2870 
- #2879
- #2871 
- #2885 
- #2867
- #2883
- #2882 
- #2891

## Pick list
```
git cherry-pick 984049f290389b69948aac64bc71e5169dbcd015
git cherry-pick d09a73efcc9a72766d1f2b8ecd60a298f709f151
git cherry-pick 6834e6468fa5e0594fc95786868a631dc1c165ca
git cherry-pick 4f6c6ea9423fb4b71e6d841d09def50a0224abe6
git cherry-pick edf2862dd5361316551cc4c52f854e3470bc10d6
git cherry-pick 68f354b627af19f23d27578e8b8b2cbded9f3da8
git cherry-pick 5103dd792f5eb3b9e119674d284994d02f7e2a18
git cherry-pick 8d669842f0919cb1631c90148211a558d515a7fc
git cherry-pick 02463aea7e2ee96c977a1def65ac12e29739b453
git cherry-pick 25931f6f3385bd98b88b56952e682d00f9385548
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.0-rc1..HEAD --pretty='format:- %s'`
```
_**Important**: Updating from a Quickstart codebase less than 2.6.x is no longer supported._

_To update to a Quickstart 2.8.x release (e.g. 2.8.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x and 2.8.x and some core modules included in Drupal 9 have been removed in Drupal 10._

_For more relevant info about upgrading from 2.6.x see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes)_

_Note: This release contains a security update for the Paragraphs admin module addressing a moderately critical security vulnerability ([SA-CONTRIB-2023-049](https://www.drupal.org/SA-CONTRIB-2023-049)). Only sites that have the optional Paragraphs admin module installed are affected by this vulnerability._

## New changes since 2.8.0-rc1
- Closes #2890: Update paragraphs_admin to 8.x-1.5 for SA-CONTRIB-2023-049. (#2891)
- Closes #2732 Alternating Split Screen paragraphs don't line up in the center (#2882)
- Update drupal/webform requirement from 6.2.0-rc3 to 6.2.0 (#2883)
- Closes #1613: Add space to footer menu items (#2867)
- Closes #2884: Address ProboCI drush issues (#2885)
- Closes #2869 Linkit CKEditor5 fix for adding urls. (#2871)
- Closes #2878 Bootstrap Barrio Subtheme not D10 Compatible. (#2879)
- Closes #2868: Fix wrapping in CKEditor5. (#2870)
- Closes #2872 CKEditor 5 strips classes from div elements (#2873)
- Closes #2786: Update phonebook/directory links in global footer. (#2863)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.8.0
